Doppler.Currency/appsettings.json

```diff
@@ -42,7 +42,14 @@
       "NoCurrency": "There are no pending USD currency for that date.",
       "CurrencyCode": "MXN",
       "CurrencyName": "Peso Mexicano"
-    }
+    },
+    "TrmService": {
```

Comment:
What is the meaning of TRM?

Reply:
En colombia se denomina este número como Tasa Representativa del Mercado(TRM)
https://docs.google.com/document/d/16dEgBjKeHtqGhjMEI2DfFDjlEQJttM381nluB08kyRU/edit